### DISCUSSION TITLE
Dark Mode for Public Dashboard & AMP Page Tracking

### DISCUSSION THREAD

**Discussion by derpycoder** [plausible/analytics#715] [Anything Else]
Hey Guys,

Request is so simple, that it fits the Header.

Dark Mode for Public Dashboard would be great, and so will be tracking of AMP Pages. (Given how lightweight the tracker is)

**Comment by Vigasaurus** [comment]
The former exists, if a user isn't logged in the dashboard respects the OS-level theme setting. 

The latter has more discussion in #220
